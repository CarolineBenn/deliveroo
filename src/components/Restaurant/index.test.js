import React from 'react';
import renderer from 'react-test-renderer';

import Restaurant from '.';


test('Restaurant renders correctly', () => {
  const tree = renderer.create(
    <Restaurant />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});