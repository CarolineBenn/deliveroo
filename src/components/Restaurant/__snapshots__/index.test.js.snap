// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurant renders correctly 1`] = `
<div>
  Restaurant
</div>
`;
